sdk/eventhub/Azure.Messaging.EventHubs.Shared/src/Processor/PartitionLoadBalancer.cs

```diff
@@ -446,7 +446,13 @@ private async Task RenewOwnershipAsync(CancellationToken cancellationToken)
                     PartitionId = ownership.PartitionId,
                     LastModifiedTime = DateTimeOffset.UtcNow,
                     Version = ownership.Version
-                });
+                })
+                .ToList();
+
+            if (ownershipToRenew.Count == 0)
+            {
+                return;
+            }
 
             try
             {
```

Comment:
I think this may throw off the logic on 461.   That is intended to track all of the partitions that this instance owns, but it looks like this change reduces that to only the partitions we renewed.  Am I overlooking something?

Reply:
Good point, discovered it when writing a test :)